### PR TITLE
Update README to correct description of NPM package and electron app

### PR DESCRIPTION
**Pull request by bhufmann**
These updates were missed in recent PRs.

Signed-off-by: Bernd Hufmann <Bernd.Hufmann@ericsson.com>

### CODE REVIEW

**Review by Rodrigoplp-work — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -108,11 +108,11 @@ For example, to start the browser example app with a specific URL, one can run
 It's possible to package the repo's example application with `electron-builder`. After running `yarn` in the repo root, do:
 
 ```bash
-  $ cd electron-app
+  $ cd examples/electron
   $ yarn package
 ```
 
-  The configured Linux packaging(s) will be generated folder `electron-app/dist`
+  The configured Linux packaging(s) will be generated folder `examples/electron/dist`
```
Comment (PatrickTasse) [inline README.md:119]: generated 'in' folder?

File: README.md [hunk 2]
```diff
@@ -26,9 +26,13 @@ Prerequisites: A GitHub account (for logging into Gitpod)
 
 ## Consume the trace viewer extension from npm
 
-We plan to distribute this extension on npm.
+The **theia-trace-extension** project publishes the following packages to NPM:
 
-Availability: TBD
+* [theia-traceviewer](https://www.npmjs.com/package/theia-traceviewer): The Theia trace viewer extension. Add this package to the package.json of your Theia application.
+* [traceviewer-base](https://www.npmjs.com/package/traceviewer-base): This package contains trace management utilities for managing traces using Trace Server applications that implement the TSP
```
Comment (PatrickTasse) [inline README.md:32]: Final '.'

File: README.md [hunk 3]
```diff
@@ -26,9 +26,13 @@ Prerequisites: A GitHub account (for logging into Gitpod)
...
+* [traceviewer-react-components](https://www.npmjs.com/package/traceviewer-components): This package contains views and utilities for visualizing traces and logs via the TSP connected a Trace Server application.
```
Comment (PatrickTasse) [inline README.md:33]: connected 'to' a Trace Server application?

**Review by PatrickTasse — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (bhufmann) [reply README.md:33]: will fix it

File: README.md (hunk 1 above)
Comment (bhufmann) [reply README.md:119]: Done

File: README.md (hunk 2 above)
Comment (bhufmann) [reply README.md:32]: Done

**Review by PatrickTasse — APPROVED** [review]

**Comment by ebugden** [comment]
I'm post merge, but looks good to me!

**Pull request merged**